include/hdlConvertor/hdlObjects/iHdlExpr.h

```diff
@@ -41,25 +53,30 @@ class iHdlExpr: public WithPos, public iHdlObj {
 	static std::unique_ptr<iHdlExpr> TYPE_T();
 	static std::unique_ptr<iHdlExpr> AUTO_T();
 
-	static std::unique_ptr<iHdlExpr> INT(int64_t val);
-	static std::unique_ptr<iHdlExpr> INT(const std::string &strVal, int base);
-	static std::unique_ptr<iHdlExpr> INT(const std::string &strVal, int bits,
+	static std::unique_ptr<iHdlExpr> INT(TerminalNode *node, int64_t val);
```

Comment:
It seems to me that the problem is that `iHdlExpr::INT` should be actually `class HdlExprInt: iHdlExpr;` and then the `create_object` function should be used instead of polluting the hdlObjects with ANTLR.
However it is not critical.

Reply:
Agreed. The HdlExpr hierarchy needs some work. I wanted to keep this commit focused on specific task rather than trying to redesign the whole thing. There are some comments (not by me) that does talk about some redesign.